pkg/piecestore/psserver/psdb/psdb.go

```diff
@@ -205,6 +208,32 @@ func (db *DB) Migration() *migrate.Migration {
 					`CREATE INDEX IF NOT EXISTS idx_bwa_serial ON bandwidth_agreements (serial_num)`,
 				},
 			},
+			{
+				Description: "Clean pieces stored using ttl",
```

Comment:
this looks like it is just deleting all the files at db.dbPath. Is it supposed to? Why?

Reply:
You are right. The requirement is to just clean up the old folder which stored pieces, to release space and leave the 'ttl' db which will cleaned up later.